packaging/create_rpm

```diff
@@ -217,68 +220,57 @@ def get_package_urls_file(edition, branch, repo_destination_path):
     }
 
 
-def download_file(source, destination):
-    """
-        Download a specific file.
-    """
-    # Please note that this function is used under multiprocessing.
-    # Therefore, if you change from print to e.g. a logger, or even if you
-    # change any prints to be multi-line, you will probably find that the
-    # output will become very... artistic.
-    print('Downloading {url}...'.format(url=source))
-    result = subprocess.check_output(
-        [
-            'curl', '--retry', '10', '--fail', '--silent',
-            '--show-error', '--location',
-            '-o', destination,
-            source,
-        ],
-        stderr=subprocess.STDOUT,
-    )
-    print('...finished downloading {url}'.format(url=source))
-    return result
+def file_downloader(to_download, log_lock):
```

Comment:
Let's maybe change `to_download` to something more descriptive, like `task_queue` or `download_queue` or whatever